pytorch_blade/src/ltc/disc_compiler/disc_compiler.cpp

```diff
@@ -87,14 +87,19 @@ void EnhancementInputShape(
     const auto ts_data =
         std::static_pointer_cast<torch::lazy::TSData>(argument);
     if (ts_data->HasValue()) {
-      auto t = ts_data->data();
-      size_t rank = t.sizes().size();
-      input->setType(c10::TensorType::create(
-          t.scalar_type(),
-          t.device(),
-          c10::SymbolicShape(c10::optional<size_t>(rank)),
-          c10::VaryingShape<c10::Stride>(rank),
-          t.requires_grad()));
+      if (torch::blade::env::ReadBoolFromEnvVar(
+              "TORCH_DISC_DYNAMIC_SHAPE_COMPILE", false)) {
```

Comment:
set default to true?

Reply:
that we need to change the var name to `TORCH_DISC_DYNAMIC_STATIC_COMPILE `, maybe update it in the future, I need more test on whole bert training graph.